tiatoolbox/wsicore/wsireader.py

```diff
@@ -36,6 +37,8 @@
 Bounds = Tuple[Number, Number, Number, Number]
 IntBounds = Tuple[int, int, int, int]
 Resolution = Union[Number, Tuple[Number, Number], np.ndarray]
+MIN_NGFF_VERSION = Version("0.4")
+MAX_NGFF_VERSION = Version("0.4")
```

Comment:
is this not quite strict?

Reply:
Yes, intentionally. It's not been tested with anything other than 0.4 so we cannot say we suppport it

Reply:
Besides, support for new versions is quite far beyond the scope of this PR.